docs/hdfs_connector.rst

```diff
@@ -95,13 +150,17 @@ You should see the following output::
 
 .. note:: If you want to run the Quickstart with Hive integration, before starting the connector,
```

Comment:
Seems like this could be a new section, since the quickstart seems like it finished the main demo and is now backtracking to describe what's different when using Hive.

Also, should the quickstart show how to use the CLI to stop the services?